lib/book_settings.py

```diff
@@ -144,12 +143,7 @@ def search_for_asin(self, query):
         except Exception as e:
             try:
                 self._aConnection.close()
-                if self._proxy:
-                    self._aConnection = HTTPSConnection(self._https_address, self._https_port)
-                    self._aConnection.set_tunnel('www.amazon.com', 443)
-                else:
-                    self._aConnection = HTTPSConnection('www.amazon.com')
-
+                self._aConnection.connect()
```

Comment:
yeah i have no idea why.. we can investigate some more

On Fri, Jun 24, 2016 at 3:21 PM Anthony Toole <notifications@github.com>
wrote:

> In lib/book_settings.py
> <https://github.com/szarroug3/X-Ray_Calibre_Plugin/pull/54#discussion_r68456799>
> :
>
> > @@ -144,12 +143,7 @@ def search_for_asin(self, query):
> >          except Exception as e:
> >              try:
> >                  self._aConnection.close()
> > -                if self._proxy:
> > -                    self._aConnection = HTTPSConnection(self._https_address, self._https_port)
> > -                    self._aConnection.set_tunnel('www.amazon.com', 443)
> > -                else:
> > -                    self._aConnection = HTTPSConnection('www.amazon.com')
> > -
> > +                self._aConnection.connect()
>
> OK - odd that it helps doing it manually (vs. the auto-close) but if it
> has a benefit then keep it. Smaller/simpler now anyway.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/szarroug3/X-Ray_Calibre_Plugin/pull/54/files/89c7d995442049dfa337e6394eb310d53c334dbd#r68456799>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIEjUdW2O8f58YeGq0OcNXyxOouUKY19ks5qPDw4gaJpZM4I90sF>
> .
>